eofConvention.fsx

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env fsx
```

Comment:
let's now make a new test that only allows two shebangs:

* this one above, with fsx
* or this one with dotnet fsi: https://twitter.com/Tarmil_/status/1361598041992687618
* (all other shebangs should be flagged as invalid)